google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/stub/EnhancedBigtableTableAdminStub.java

```diff
@@ -52,7 +53,7 @@ public class EnhancedBigtableTableAdminStub extends GrpcBigtableTableAdminStub {
   private final BigtableTableAdminStubSettings settings;
   private final ClientContext clientContext;
 
-  private final AwaitReplicationCallable awaitReplicationCallable;
```

Comment:
you cant just remove this callable, it'll break public apis. Please make this a delegate for the new callable